packages/ui/src/Card.module.scss

```diff
@@ -0,0 +1,47 @@
+@use '../styles/constants' as c;
+
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  align-content: stretch;
+  box-shadow: 0 c.$cardShadowSize c.$cardShadowBlur 0 c.$colorShadow;
+  border-radius: c.$borderRadius;
+
+  &.typeSecondary {
+    background: none;
+    border: c.$borderWidth solid c.$colorNeutralLight;
+    box-shadow: none;
+  }
+
+  &.typeHighlighter {
+    background: c.$colorInfoLight;
+    box-shadow: none;
+  }
+
+  .header {
+    padding: c.$grid * 6 c.$grid * 6 c.$grid * 3;
+
+    h2,
+    h3 {
+      display: flex;
+      align-items: center;
+      padding: 0;
+      margin: 0;
+
+      > svg,
+      > button {
+        margin-left: c.$grid * 2;
+      }
+    }
+  }
+
+  .content {
+    @include c.typographyBodyNormal;
+
+    padding: c.$grid * 3 c.$grid * 6 c.$grid * 6;
+
+    &.woTitle {
```

Comment:
nit: What does this `wo` prefix stand for? `without`? I think it'd be more readable to use that.

Reply:
We can get rid of this class if we let `.content` handle the padding at all times:

Alter `.content`: 
from `padding: c.$grid * 3 c.$grid * 6 c.$grid * 6;` to `padding: c.$grid * 6;`

Alter `.header`:
from `padding: c.$grid * 6 c.$grid * 6 c.$grid * 3;` to `padding: c.$grid * 6 c.$grid * 6 0`

Reply:
Even better!

Reply:
It actually "resolves" both this and my comment, we gotta keep this open. :)

Reply:
@jankoritak I don't know why, but for some reason I can't reply to your comment :D Good idea, I'll apply your suggestion 👍 
![image](https://user-images.githubusercontent.com/25319634/99181971-248cfe00-2732-11eb-9388-9969ff855a35.png)